repository/project.go

```diff
@@ -117,7 +117,7 @@ func (r *ProjectRepository) DeleteProject(ctx context.Context, id uuid.UUID) err
 	}
 
 	if result.RowsAffected() == 0 {
-		return apierrors.NewProjectNotFoundError().Wrap(err)
+		return apierrors.NewProjectNotFoundError()
```

Comment:
Just tweaking changes from previous PR. There is no error to wrap (error is nil that point).